scripts/py_matter_yamltests/matter_yamltests/runner.py

```diff
@@ -122,14 +122,6 @@ def run(self, config: TestRunnerConfig) -> bool:
 
 
 class TestRunner(TestRunnerBase):
-    """
-    TestRunner is a default runner implementation.
```

Comment:
Why is the documentation for the class itself being removed?

Reply:
The documentation seemed to have been added as a filler - the member is gone and the documentation is saying "this is a test runner implementation". The fact that this is a default seems to maybe be non-trivial, however it still does not seem to help.

Generally see additions in #28704 which I update in this PR with a different approach which I find more sane.

Reply:
OK.